NuGet.config

```diff
@@ -3,6 +3,7 @@
   <packageSources>
     <clear />
     <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
```

Comment:
@mmitche @wtgodbe where are we on `dotnet-core` versus `dotnet-eng` versus `dotnet-tools`? Do we need two of those three these days?

Reply:
We can remove `dotnet-tools` in favor of `dotnet-eng`. There might be some straggling packages still in `dotnet-core` that prevent us from removing it - rel/3.1 still has it:

https://github.com/dotnet/aspnetcore/blob/64a7f37e8afb060f3dbf8a679e1f7112a4558562/NuGet.config#L14

Reply:
dotnet-core can probalby be removed in favor of "dotnet5" and "dotnet5-transport"

Reply:
(fyi I removed it from websdk recently)